cdap-app-fabric/src/main/java/co/cask/cdap/adapter/AdapterSpecification.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright © 2015 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.adapter;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Specification that is used to configure an adapter.
+ */
+public final class AdapterSpecification {
+
+  private final String name;
+
+  private final AdapterType type;
+
+  private final Map<String, String> properties;
+
+  private final List<Source> sources;
```

Comment:
what does it mean for an adapter to have multiple sources and multiple sinks? Does it mean the first source goes to the first sink, the second source goes to the second sink and so on?